s3transfer/manager.py

```diff
@@ -27,26 +29,40 @@
 class TransferConfig(object):
     def __init__(self,
                  multipart_threshold=8 * MB,
-                 max_concurrency=10,
                  multipart_chunksize=8 * MB,
-                 max_queue_size=0,
+                 max_request_concurrency=10,
```

Comment:
Given this a backwards incompatible change, what's the plan to interop with the boto3 transfer config object?